system/processes/README.md

```diff
@@ -0,0 +1,54 @@
+# SYSTEM GENERIC SignalFx detectors
+
+## How to use this module
+
+```hcl
+module "signalfx-detectors-system-processes" {
+  source      = "github.com/claranet/terraform-signalfx-detectors.git//system/processes?ref={revision}"
+
+  environment = var.environment
+  notifications = var.notifications
+}
+
+```
+
+## Purpose
+
+Creates SignalFx detectors with the following checks:
+
+- Processes aliveness
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| processes\_aggregation\_function | Aggregation function and group by for processes detector (i.e. ".mean(by=['host']).") | `string` | `""` | no |
+| processes\_disabled | Disable all alerting rules for processes detector | `bool` | n/a | yes |
+| processes\_disabled\_critical | Disable critical alerting rule for processes detector | `bool` | n/a | yes |
+| processes\_disabled\_warning | Disable warning alerting rule for processes detector | `bool` | n/a | yes |
+| processes\_notifications | Notification recipients list for every alerting rules of processes detector | `list` | `[]` | no |
+| processes\_notifications\_critical | Notification recipients list for critical alerting rule of processes detector | `list` | `[]` | no |
+| processes\_notifications\_warning | Notification recipients list for warning alerting rule of processes detector | `list` | `[]` | no |
+| processes\_threshold\_critical | Critical threshold for processes detector | `number` | `1` | no |
+| processes\_threshold\_warning | Warning threshold for processes detector | `number` | `2` | no |
+| processes\_transformation\_function | Transformation function for processes detector (mean, min, max) | `string` | `"min"` | no |
+| processes\_transformation\_window | Transformation window for processes detector (i.e. 5m, 20m, 1h, 1d) | `string` | `"1h"` | no |
+| detectors\_disabled | Disable all detectors in this module | `bool` | `false` | no |
+| environment | Infrastructure environment | `string` | n/a | yes |
+| filter\_custom\_excludes | List of tags to exclude when custom filtering is used | `list` | `[]` | no |
+| filter\_custom\_includes | List of tags to include when custom filtering is used | `list` | `[]` | no |
+| notifications | Notification recipients list for every detectors | `list` | n/a | yes |
+| prefixes | Prefixes list to prepend between brackets on every monitors names before environment | `list` | `[]` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| processes\_id | id for detector processes |
+
+## Related documentation
+
+[Official documentation for processes](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-processes.html)
+
+## Notes
```

Comment:
could be deleted if not useful but do not hesitate to provide a configuration sample ;)

Reply:
I fill it up